### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Ugboaja-Uchechi**
### Project Requirements

- Create a database named vet_clinic.

- Create a table named animals with the following columns:
  id: integer
  name: string
  date_of_birth: date
  escape_attempts: integer
  neutered: boolean
  weight_kg: decimal

- Copy the SQL you used in the previous point into the schema.sql file in the generated repository.

- Insert the following data:
  Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
  Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
  Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
  Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
-Copy the SQL you used in the previous point into the data.sql file in the generated repository.

- Write queries for the following:
  Find all animals whose name ends in "mon".
  List the name of all animals born between 2016 and 2019.
  List the name of all animals that are neutered and have less than 3 escape attempts.
  List date of birth of all animals named either "Agumon" or "Pikachu".
  List name and escape attempts of animals that weigh more than 10.5kg
  Find all animals that are neutered.
  Find all animals not named Gabumon.
  Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
- Copy the SQL you used in the previous point into the queries.sql file in the generated repository.

### CODE REVIEW

**Comment by Ugboaja-Uchechi** [comment]
I have worked on all project requirements to the best of my knowledge.

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```
Comment (AshakaE) [inline queries.sql]: This query will target for 'mon' in between word, what we are targeting is for words that end with 'mon'
- [x] Please update this query

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Ugboaja-Uchechi 🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Stephanie, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**